src/ClientBuilder.php

```diff
@@ -231,13 +239,29 @@ public function setRepresentationSerializer(RepresentationSerializerInterface $r
     /**
      * {@inheritdoc}
      */
-    public function setSdkIdentifierSuffix(?string $sdkIdentifierSuffix): void
+    public function setSdkIdentifier(string $sdkIdentifier): void
     {
-        $this->sdkIdentifier = Client::SDK_IDENTIFIER;
+        $this->sdkIdentifier = $sdkIdentifier;
+    }
 
-        if ($sdkIdentifierSuffix) {
-            $this->sdkIdentifier .= '.' . $sdkIdentifierSuffix;
+    /**
+     * @return string
+     */
+    private function getSdkVersion(): string
+    {
+        if (null === $this->sdkVersion) {
+            $this->sdkVersion = PrettyVersions::getVersion('sentry/sentry')->getPrettyVersion();
```

Comment:
This is just a lazy way of fetching the default value. Notice that his getter is private, it's just an internal implementation detail.